CITATION.cff

```diff
@@ -0,0 +1,86 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it using these metadata.
+title: 'GEFF: Graph Exchange File Format'
+authors:
+  - given-names: Morgan
```

Comment:
@msschwartz21 please double check your details.

Reply:
Perfect! Thanks for putting this together!